### PR TITLE
Feature/lfo/op 189 ies publisher

### PR DESCRIPTION
**Pull request by lfoIncluit**

### CODE REVIEW

File: BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp [hunk 1]
```diff
@@ -0,0 +1,210 @@
+// Copyright (c) 2019 Intel Corporation.
```
Comment (mvasquez81) [inline BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp]: 2020, if the date represents the last year edition

File: BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        // Copy over the string
+        size_t len = strlen(body->body.string);
+        m_message = new char[len + 1];
+        memcpy(m_message, body->body.string, len);
+        m_message[len] = '\0';
+        msgbus_msg_envelope_destroy(msg);
+    };
+
+    /**
+     * Destructur
```
Comment (mvasquez81) [inline BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp]: typo: Destructor

File: BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/subscriber_thread.cpp [hunk 3]
```diff
@@ -33,45 +33,108 @@
...
+        // Copy over the string
+        size_t len = strlen(body->body.string);
+        m_message = new char[len + 1];
+        memcpy(m_message, body->body.string, len);
+        m_message[len] = '\0';
+        msgbus_msg_envelope_destroy(msg);
+    };
 
     /**
      * Destructur
```
Comment (mvasquez81) [inline BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/subscriber_thread.cpp]: typo

File: BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+int main(int argc, char** argv) {
+    if(argc == 1) {
+        LOG_ERROR_0("Too few arguments");
+        return -1;
+    } else if(argc > 2) {
+        LOG_ERROR_0("Too many arguments");
+        return -1;
+    }
```
Comment (mvasquez81) [inline BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp]: Both error cases could be unified and print the usage too.

File: BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp (hunk 1 above)
Comment (lfoIncluit) [reply BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp:1]: example code, not related to BlindSpot execution.

Comment (lfoIncluit) [reply BlindspotAssistance/common/eis_common/libs/EISMessageBus/examples/pub_thread.cpp]: fixed

**Review by mvasquez81 — DISMISSED** [review]

**Review by mvasquez81 — APPROVED** [review]

**Review by mvasquez81 — APPROVED** [review]

**Pull request merged**